src/views/Player.vue

```diff
@@ -18,12 +18,29 @@
       v-if="isQuestionShown"
       data-test="modal"
     ></QuestionModal>
+
```

Comment:
Comparing with Plio - why have you changed the following:
- used `v-if` instead of `isShown` (the hiding of the scorecard is already happening with `:class="{
        hidden: !isScorecardShown,
      }"`
- why have you removed `greeting` as a prop?
<img width="966" alt="Screenshot 2022-04-07 at 11 18 20 AM" src="https://user-images.githubusercontent.com/13185699/162129220-a2719819-bff3-4731-b5c5-5a115faa3c6a.png">